src/js/angular/ttyg/templates/modal/agent-settings-modal.html

```diff
@@ -134,8 +134,12 @@ <h4 class="modal-title">{{(isEdit ? 'ttyg.agent.create_agent_modal.title.edit '
 
                         <div ng-if="extractionMethod.method === extractionMethods.SIMILARITY"
                              class="extraction-method-options">
-                            <div ng-if="!similarityIndexes.length" class="no-similarity-index-message"
-                                 ng-bind-html="getNoSimilarityIndexHelpMessage()">
+                            <div ng-if="!similarityIndexes.length" class="no-similarity-index-message">
+                                {{'ttyg.agent.create_agent_modal.form.similarity_index.no_similarity_index_message.message_prefix' | translate}}
```

Comment:
I don't like splitting these messages, but I guess that the link with the angular directive doesn't work if you put it in the label. Right?